docs/src/markdownDocs/themes/overview.mdx

```diff
@@ -0,0 +1,56 @@
+import { CodeBlock, CallToActionButton, ThemeExplorer } from '../../shared';
+import { PlayArrow, Code, BugReport, Difference } from '@mui/icons-material';
+
+# React Themes Overview
+
+This package, `@brightlayer-ui/react-themes`, provides theming support for Eaton applications using the Brightlayer UI design system. It includes resources for developers using React with [MUI's Material UI](https://www.npmjs.com/package/@mui/material) version 5+. This package comes with two theme options: a Blue theme (standard) and a Dark theme.
+
+<CallToActionButton
+    title={`Style Guide: Themes`}
+    description={`Follow Brightlayer UI's design guideline.`}
+    url={`https://brightlayer-ui.github.io/style/themes`}
+/>
+<CallToActionButton
+    title={`Example Project`}
+    description={`An interactive "kitchen sink" demo project, where both Material UI and Brightlayer UI components are rendered in Brightlayer UI themes.`}
+    url={`https://blui-react-showcase.web.app/`}
+    avatar={<PlayArrow sx={{ height: 48, width: 48 }} />}
+/>
+<CallToActionButton
+    title={`Source Code`}
+    description={`Examine the source code on GitHub.`}
+    url={`https://github.com/etn-ccis/blui-react-themes/tree/master`}
+    avatar={<Code sx={{ height: 48, width: 48 }} />}
+/>
+<CallToActionButton
+    title={`Submit an Issue`}
+    description={`Log a bug or request for a new feature on GitHub.`}
+    url={`https://github.com/etn-ccis/blui-react-themes/issues`}
+    avatar={<BugReport sx={{ height: 48, width: 48 }} />}
+/>
+<CallToActionButton
+    title={`Change Log`}
+    description={`Learn what’s new in the latest version.`}
+    url={`https://github.com/etn-ccis/blui-react-themes/blob/master/CHANGELOG.md`}
+    avatar={<Difference sx={{ height: 48, width: 48 }} />}
+/>
+
+# What Do The Themes Look Like?
+
+Have a look at the component theme explorer below and learn how Brightlayer UI themes work at a high level. Here we showcase some of the most representative components from Material Design and Brightlayer UI, and how our themes affect them.
+
+<ThemeExplorer />
+
+You are also encouraged to explore our more detailed [theme example project](https://blui-react-showcase.web.app/).
```

Comment:
```suggestion
You may find it beneficial to explore our more detailed [showcase project](https://blui-react-showcase.web.app/) which gives a clearer picture of the Material Design and Brightlayer UI components and theme interactions.
```

Reply:
```suggestion
You are also encouraged to explore our more detailed [example / showcase project](https://blui-react-showcase.web.app/) which gives a clearer picture of the Material Design and Brightlayer UI components and theme interactions.
```

I want to stay consistent with the verbiage used on the workflow documentation's side (to be implemented). The Workflow docs call its example project "example". So I think it is better to mention the word "example" if we want to avoid potential confusion with our adopter users.